FFMpegCore.Test/TasksExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FFMpegCore.Test
+{
+    static class TasksExtensions
+    {
+        public static T WaitForResult<T>(this Task<T> task)
+        {
+            task.Wait();
```

Comment:
maybe `task.ConfigureAwait(false).GetAwait().GetResult()` is better?

Reply:
Sure